tree/dataframe/src/RDataFrame.cxx

```diff
@@ -677,8 +677,22 @@ thread-safety, see [here](#generic-actions).
 <a name="reference"></a>
 */
 // clang-format on
+
 namespace ROOT {
 
+namespace Detail {
+namespace RDF {
+class RCustomColumnBase;
+}
+} // namespace Detail
+
+using RCustomColumnBasePtrMap_t = std::map<std::string, std::shared_ptr<ROOT::Detail::RDF::RCustomColumnBase>>;
+using RCustomColumnBasePtrMapPtr_t = std::shared_ptr<const RCustomColumnBasePtrMap_t>;
+using ColumnNames_t = ROOT::Detail::RDF::ColumnNames_t;
+using ColumnNamesPtr_t = std::shared_ptr<const ColumnNames_t>;
+
+namespace RDFInternal = ROOT::Internal::RDF;
```

Comment:
If I'm not mistaken this is importing `RDFInternal` in `namespace ROOT`. we should avoid polluting our main namespace.

now that I think about it, I'm not sure why `RDataFrame` is able to use `RDFDetail`, that should also not be available in this scope

Reply:
Fixed 9e77537bdefd4c693d469e8820ddf8fc92766cce